catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/DataListFragment.java

```diff
@@ -268,6 +266,13 @@ private void deleteItems(List<UserData> selectedItems) {
 				selectedItems.size()));
 	}
 
+	private boolean isItemUsed(List<UserData> selectedItems){
+		//check if items are useed
+		//if one of them being used - true, otherwise - false
```

Comment:
Yep, it seems rather logical to move it into UserData, I'll check this out. About returning only one boolean from the list of arguments, it is done in case of deleting a list of items all at once by the user through the menu in the top right corner.